.github/workflows/quicktest.yaml

```diff
@@ -52,11 +57,19 @@ jobs:
 
   # Run FastSurfer on MRI data
   run-fastsurfer:
-    runs-on: ci-gpu
+    runs-on: self-hosted
     needs: build-singularity-image
     steps:
     - name: Run FastSurfer
       run: |
+        cd $RUNNER_FS_OUTPUT
```

Comment:
We are going to have 3 subjects for the quicktest, please put the subject names into a subject_list file as used by brun_fastsurfer and also use brun_fastsurfer here to run all subjects from there.

Also create a new output folder (--sd) based on the current git hash.

You can add things like `--parallel_subjects` for `brun_fastsurfer` to optimize the processing.